### PR TITLE
Feature/confirm opt in utl with html page

### PR DESCRIPTION
**Pull request by TranHungKT**

### CODE REVIEW

File: static/base.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  border: 1px solid #F8F9FA;
+  border-radius: 8px;
+  box-shadow: 6px 1px 17px 0px #F8F9FA;
+  -webkit-box-shadow: 6px 1px 17px 0px #F8F9FA;
+  -moz-box-shadow: 6px 1px 17px 0px #F8F9FA;
+}
+
+.body {
+  background-color: #F8F9FA ;
+}
```
Comment (TranHungKT) [inline static/base.css]: Please fix this

File: static/public/header.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Inter:400,600" rel="stylesheet">
+  <link href="http://localhost:3000/static/base.css" rel="stylesheet" type="text/css" />
+  <link href="http://localhost:3000/static/style.css" rel="stylesheet" type="text/css" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
+
+  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
+  <link rel="shortcut icon" href="http://localhost:3000/static/images/favicon.png" type="image/x-icon" />
+</head>
+<body class="body">
+{{end}}
```
Comment (TranHungKT) [inline static/public/header.html]: And this

File: static/style.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.logo {
+  border-bottom: 1px solid black; 
+  padding-bottom: 10px;
+}
+
+.confirm-optin-container {
+  padding-top: 20px;
+}
```
Comment (TranHungKT) [inline static/style.css]: This one too

File: controllers/subscriberControllers/confirmOptinController.go [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+			context.TODO(),
+			bson.D{bson.E{Key: "email", Value: email}},
+			bson.M{"$set": bson.M{"lists.$[list].subscriptionStatus": models.SubscriptionStatusConfirmed}}, &filterOption).Decode(&updatedDocument)
+
+		if err != nil {
+			ctx.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+			return
+		}
+
+		ctx.HTML(http.StatusOK, "confirmOptin.html", gin.H{
```
Comment (TranHungKT) [inline controllers/subscriberControllers/confirmOptinController.go]: Seem like this one is a mistake

File: controllers/subscriberControllers/createNewSubscriberController.go [hunk 5]
```diff
@@ -33,7 +33,7 @@ func CreateNewSubscriberController() gin.HandlerFunc {
 			return
 		}
 
-		if count > 0 {
+		if count > 10 {
```
Comment (TranHungKT) [inline controllers/subscriberControllers/createNewSubscriberController.go]: And this one

File: utils/email.go [hunk 6]
```diff
@@ -20,14 +20,14 @@ const (
 type Request struct {
 	to      []string
 	subject string
-	body    string
+	Body    string
```
Comment (TranHungKT) [inline utils/email.go]: Why we need these changes?

File: utils/email.go [hunk 7]
```diff
@@ -41,15 +41,15 @@ func (r *Request) SendEmailsByGoMail() {
...
+	mail.SetBody("text/html", r.Body)
 	if err := d.DialAndSend(mail); err != nil {
 		fmt.Print(err)
 		panic(err)
 	}
 
 }
 func (r *Request) ParseTemplate(templateFileName string, data interface{}) error {
-	t, err := template.ParseFiles(templateFileName)
+	t, err := template.ParseFiles("/Users/tranhung/Documents/email-management/static/public/successOptin.html", templateFileName)
```
Comment (TranHungKT) [inline utils/email.go]: And this one too, seem like it should not be changed

**Review by ngocanhtranhi — APPROVED** [review]

**Pull request merged**